### PR TITLE
OpenBabel with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

**Comment by boegel** [comment]
@RvDijk missing dep?

```
EasyBuildError: 'Irresolvable dependencies encountered: Eigen/3.2.8-foss-2016a'
```

If this is provided via another PR, please clarify that in the description

**Comment by boegel** [comment]
other than the missing dep: lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2496.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cc74554d46e820dc03ba7ab2081a2401 for a full test report.

**Comment by boegel** [comment]
@RvDijk one of the tests is failing for me, did it work for you?

```
The following tests FAILED:
	 10 - rotor_Test (Failed)
Errors while running CTest
make: *** [test] Error 8
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2150.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/52626da0e62f88ff2c0212688538371c for a full test report.

**Comment by RvDijk** [comment]
@boegel the installation did not fail for me. Do you still have the error with the Eigen module included?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2444.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0b05d684c9d6c6161490addb630d0a22 for a full test report.

**Comment by boegel** [comment]
@RvDijk still fails...

**Comment by RvDijk** [comment]
@boegel, I'm not able to reproduce the failed test on CentOS6. I found several places reporting this bug (the issue seems to be numerical rounding) e.g.:
http://forums.openbabel.org/rotor-test-fails-on-AArch64-architecture-td4657545.html

Here is a patch that might be able to solve for the issue. 
https://sourceforge.net/p/openbabel/bugs/945/

Do you have any suggestions to make this work?

**Comment by boegel** [comment]
@RvDijk one option would be to try and see if using more strict floating-point precision helps

You can enable this using `strict: True` in the `toolchainopts` parameter, see also http://easybuild.readthedocs.io/en/develop/version-specific/toolchain_opts.html#foss

**Comment by boegel** [comment]
@RvDijk please sync this branch with `develop`, since we already have `Eigen-3.2.8-foss-2016a.eb` included there

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2539.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e8dca355fc9a7dab3e3f04a893924a6d for a full test report.

**Comment by boegel** [comment]
@RvDijk same test is failing again... :(

Is applying the patch you found an option?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2130.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8093d33805fc3753780ba6e105ba01d8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2427.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/65fe1f1306f92407e7b5957e931706ce for a full test report.

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-2.3.2-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'strict': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = [
+    'OpenBabel-%(version)s-fix-link-path-tests.patch',
+    'OpenBabel-%(version)s-ignore-failed-test.patch',
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-2.3.2-foss-2016a-Python-2.7.11.eb]: @RvDijk you need to list the patch here to make sure it gets applied...

Comment (boegel) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-2.3.2-foss-2016a-Python-2.7.11.eb]: Can you also rename it similar to the existing patches?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2427.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1913aceea161ad1172503c517a4dd9ab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2062.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ba3b0df9d6cc247399c4ddab43647972 for a full test report.

**Comment by boegel** [comment]
@RvDijk patch doesn't help?

**Comment by RvDijk** [comment]
@boegel apparently so... I can't think of more solutions for this test...

**Comment by boegel** [comment]
@RvDijk since there clearly are problems with that test (that are not getting resolved, it seems), how about using a patch to skip/remove that test for now?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2647.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bb64597c58262b045a566308c1d93d74 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2112.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e596629e94efafc98262c9ea9d74a64a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**